src/Nethermind/Nethermind.Runner/configs/linea-mainnet_archive.cfg

```
@@ -0,0 +1,18 @@
+{
+  "Init": {
+    "ChainSpecPath": "chainspec/linea-mainnet.json",
+    "GenesisHash": "0xb6762a65689107b2326364aefc18f94cda413209fab35c00d4af51eaa20ffbc6",
+    "BaseDbPath": "nethermind_db/linea-mainnet_archive",
+    "LogFileName": "linea-mainnet_archive.logs.txt"
+  },
+  "Merge": {
+    "Enabled": false
+  },
```

Archive also needs pruning off?
```suggestion
  },
  "Pruning": {
    "Mode": "None"
  },
```